nucliadb_utils/nucliadb_utils/audit/stream.py

```diff
@@ -229,6 +283,22 @@ async def search(
         auditrequest.trace_id = get_trace_id()
         await self.send(auditrequest)
 
+        self.kb_usage_utility.send_kb_usage(
+            service=Service.NUCLIA_DB,
+            account_id=None,
+            kb_id=kbid,
+            kb_source=KBSource.HOSTED,
+            # TODO unify AuditRequest client type and Nuclia Usage client type
```

Comment:
should this comment be removed? or are you planning on unifying some time soon?

Reply:
I will unify it soon